### DISCUSSION TITLE
我放了只小马在这里能帮我照看一下吗

### DISCUSSION THREAD

**Discussion by kzjxb** [share121/inter-knot#3053] [其他]
![微信图片_20240724171107](https://github.com/user-attachments/assets/7ee7b837-268c-4a9f-8867-96a4266eb1a3)
千万别做坏事哦~

**Comment by homteam** [comment]
曼波<img src="https://img-static.mihoyo.com/communityweb/upload/23cbc98f6908928ed4b4495421310664.png" alt="偶像姬_糖葫芦" width="75" >

**Comment by yivitte** [comment]
一把抓住顷刻炼化

**Comment by BlueOrange2049** [comment]
这个单我接了
